app/controllers/solidus_bolt/accounts_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module SolidusBolt
+  class AccountsController < BaseController
+    def create
+      user = Spree::User.find_by(email: params[:email])
+      if user
+        render json: {}, status: :ok
+      else
+        render json: {}, status: :not_found
```

Comment:
can we use instead use the following ?
```rb
render head :not_found
```

Reply:
see above

Reply:
modified in https://github.com/nebulab/solidus_bolt/commit/cbad318944309509197f3931924794104865de47